src/verifier.ts

```diff
@@ -65,7 +79,8 @@ export class Verifier extends Engine<VerifyProgress> {
     // Get package info
     let info: any;
     try {
-      info = await this.get(`https://registry.npmjs.com/${packageName}`);
+      let path = scope != undefined ? `${scope}/${packageName}` : `${packageName}`;
```

Comment:
I think this could be `const` instead of `let`.

Reply:
This could be handled by the tslint `prefer-const` auto-fixer. Maybe we can take the chance and create a `lint-fix` npm script to get this one and all future cases covered.

Reply:
That sounds like a great idea. But let's do it in a separate PR :+1:

Reply:
Fixed here, lint-fix is in #9